src/ui/screens/Council/CouncilScreen.tsx

```diff
@@ -330,7 +330,7 @@ function SubmitCandidacyModel({visible, setVisible, moduleElection}: SubmitCandi
             method: `${moduleElection.module}.submitCandidacy`,
             params: [argsLength === 1 ? [council?.candidates.length ?? 0] : []],
           },
-        });
+        }).then(() => refetchCouncil);
```

Comment:
The function is not called.

```suggestion
        }).then(() => refetchCouncil());
```